tests/test_features.nim

```diff
@@ -0,0 +1,29 @@
+# Copyright 2023 Status Research & Development GmbH
+# Licensed under either of
+#
+#  * Apache License, version 2.0, ([LICENSE-APACHE](LICENSE-APACHE) or http://www.apache.org/licenses/LICENSE-2.0)
+#  * MIT license ([LICENSE-MIT](LICENSE-MIT) or http://opensource.org/licenses/MIT)
+#
+# at your option. This file may not be copied, modified, or distributed except according to those terms.
+
+import ../stint, unittest
+
+suite "new features":
+  test "custom literal":
+    const
+      a = 0xabcdef0123456'u128
```

Comment:
You should add some tests with longer strings (i.e. numbers outside the uint64 range).
Also, the code should produce a compilation error if the string is too long (outside the Stint type range).

Reply:
fixed